include/eve/module/math/regular/horner.hpp

```diff
@@ -57,12 +57,12 @@ namespace eve
 //!   namespace eve
 //!   {
 //!      // Regular overloads
-//!      constexpr auto horner(value auto x, value auto ...ci)                         noexcept; // 1
+//!      constexpr auto horner(value auto x, value auto ci...)                         noexcept; // 1
```

Comment:
This commit just revert all the fixes from before, be careful.

Reply:
done